git-buildkite

```diff
@@ -120,6 +120,14 @@ function current_head_dirty() {
   [ "$(git status --short)" ]
 }
 
+function protected_branches() {
+  git config buildkite.protectedBranches 2> /dev/null | tr ',' '\n'
+}
+
+function is_protected_branch() {
+  protected_branches | grep -q "$1"
```

Comment:
`grep -q -- "$1"` just in case the branch starts with a `-`? (unlikely, I know...)